lib/segment/src/index/hnsw_index/hnsw.rs

```diff
@@ -798,6 +806,7 @@ impl HNSWIndex {
         params: Option<&SearchParams>,
         custom_entry_points: Option<&[PointOffsetType]>,
         vector_query_context: &VectorQueryContext,
+        hw_counter: &HardwareCounterCell,
```

Comment:
`vector_query_context` already has `HardwareCounterCell`

Reply:
Thanks, missed that. Fixed now!